functions/json-loop/1.0/index.js

```diff
@@ -0,0 +1,19 @@
+const jsonLoop = async ({ input, path }, steps) => {
+  try {
+    input = JSON.parse(input);
+  } finally {
+    if (path && path.includes('.')) {
+      path.split('.').forEach((key) => {
+        input = input[key];
+      });
+    } else if (typeof path !== 'undefined' && path) {
```

Comment:
After checking again `typeof path !== 'undefined'` is not needed just check if path is there:

`switch (path) {
  case path.includes('.'): {
    path.split('.').forEach((key) => {
      input = input[key];
    });
    break;
  }

  default:
    input = input[path];
}
`